avaframe/com1DFA/com1DFA.py

```diff
@@ -1065,14 +1067,15 @@ def DFAIterate(cfg, particles, fields, dem):
     log.debug('Use standard time stepping')
     # Initialize time and counters
     nSave = 1
-    Tcpu['nSave'] = nSave
+    tCPU['nSave'] = nSave
```

Comment:
why not just switch over to do 
```suggestion
    tCPU['nSave'] = 1
```
and drop nSave?